assets/sass/widgets/_googlesitekit-widget-analyticsAllTraffic.scss

```diff
@@ -84,13 +85,19 @@
 
 			&.googlesitekit-widget--analyticsAllTraffic__dimensions--loading {
 				left: 50%;
+				margin-top: -11px;
 				position: absolute;
 				top: 50%;
 				transform: translate(-50%, -50%);
+
+				@media (min-width: $bp-desktop) {
+					margin-top: -18px;
+				}
```

Comment:
The negative margins look a bit odd here, can you explain why these are needed?

Reply:
Asvin explained that these are needed to account for the legend, which is being replaced soon anyways in #2700 so this is fine for now 👍